main/test/ca/uwaterloo/flix/library/TestNel.flix

```diff
@@ -589,210 +589,137 @@ namespace Nel {
+    @test
+    def sortBy06(): Bool = testSortByVsSort(Nel(0, 1 :: 2 :: 3 :: 4 :: 5 :: Nil))
 
-@test
-def sortBy09(): Bool = testSortByVsSort(Nel(2, 3 :: 0 :: 4 :: 1 :: 2 :: Nil))
+    @test
+    def sortBy07(): Bool = testSortByVsSort(Nel(5, 4 :: 3 :: 2 :: 1 :: 0 :: Nil))
 
-enum R {
-    case R({i: Int32, s: String})
-}
+    @test
+    def sortBy08(): Bool = testSortByVsSort(Nel(5, 3 :: 0 :: 4 :: 1 :: 2 :: Nil))
 
-instance Eq[R] {
-    pub def eq(a: R, b: R): Bool =
-        let R(x) = a;
-        let R(y) = b;
-        x.i == y.i and x.s == y.s
-}
+    @test
+    def sortBy09(): Bool = testSortByVsSort(Nel(2, 3 :: 0 :: 4 :: 1 :: 2 :: Nil))
 
-// Tests stability
-@test
-def sortBy10(): Bool =
-    Nel.sortBy(r -> let R(x) = r; x.i,
-        Nel(R({i = 1, s = "1"}),
-            R({i = 1, s = "2"}) ::
-            R({i = 1, s = "3"}) ::
-            R({i = 1, s = "4"}) ::
-            R({i = 1, s = "5"}) ::
-            R({i = 1, s = "6"}) ::
-            R({i = 1, s = "7"}) ::
-            R({i = 1, s = "8"}) ::
-            R({i = 1, s = "9"}) :: Nil)
-        )
-    ==  Nel(R({i = 1, s = "1"}),
-            R({i = 1, s = "2"}) ::
-            R({i = 1, s = "3"}) ::
-            R({i = 1, s = "4"}) ::
-            R({i = 1, s = "5"}) ::
-            R({i = 1, s = "6"}) ::
-            R({i = 1, s = "7"}) ::
-            R({i = 1, s = "8"}) ::
-            R({i = 1, s = "9"}) :: Nil)
-
-// Tests stability
-@test
-def sortBy11(): Bool =
-    Nel.sortBy(r -> let R(x) = r; x.i,
-        Nel(R({i = 5, s = "5a"}),
-            R({i = 4, s = "4a"}) ::
-            R({i = 3, s = "3a"}) ::
-            R({i = 2, s = "2a"}) ::
-            R({i = 1, s = "1a"}) ::
-            R({i = 2, s = "2b"}) ::
-            R({i = 4, s = "4b"}) ::
-            R({i = 5, s = "5b"}) ::
-            R({i = 5, s = "5c"}) :: Nil)
-        )
-    ==  Nel(R({i = 1, s = "1a"}),
-            R({i = 2, s = "2a"}) ::
-            R({i = 2, s = "2b"}) ::
-            R({i = 3, s = "3a"}) ::
-            R({i = 4, s = "4a"}) ::
-            R({i = 4, s = "4b"}) ::
-            R({i = 5, s = "5a"}) ::
-            R({i = 5, s = "5b"}) ::
-            R({i = 5, s = "5c"}) :: Nil)
-
-// Tests stability
-@test
-def sortBy12(): Bool =
-    Nel.sortBy(r -> let R(x) = r; x.i,
-        Nel(R({i = 2, s = "2a"}),
-            R({i = 1, s = "1a"}) ::
-            R({i = 3, s = "3a"}) ::
-            R({i = 3, s = "3b"}) ::
-            R({i = 1, s = "1b"}) ::
-            R({i = 3, s = "3c"}) ::
-            R({i = 2, s = "2b"}) ::
-            R({i = 2, s = "2c"}) ::
-            R({i = 2, s = "2d"}) :: Nil)
-        )
-    ==  Nel(R({i = 1, s = "1a"}),
-            R({i = 1, s = "1b"}) ::
-            R({i = 2, s = "2a"}) ::
-            R({i = 2, s = "2b"}) ::
-            R({i = 2, s = "2c"}) ::
-            R({i = 2, s = "2d"}) ::
-            R({i = 3, s = "3a"}) ::
-            R({i = 3, s = "3b"}) ::
-            R({i = 3, s = "3c"}) :: Nil)
+    enum R {
```

Comment:
I think we can get rid of the enum and instance if we're not using it anymore.

Reply:
Thanks, I had clearly missed that. Removed it.